pynest/nest/tests/test_connect_arrays.py

```diff
@@ -80,15 +134,14 @@ def test_connect_arrays_no_delays(self):
         """Connecting NumPy arrays without specifying delays"""
         n = 10
         nest.Create('iaf_psc_alpha', n)
-        sources = np.arange(1, n+1, dtype=np.uint64)
-        targets = np.arange(1, n+1, dtype=np.uint64)
-        weights = np.ones(len(sources))
-        # delays = np.ones(len(sources))
-        syn_model = 'static_synapse'
+        sources = [i for i in range(1, n + 1)]
+        targets = self.non_unique
+        weights = np.ones(n)
 
-        nest.Connect(sources, targets, syn_spec={'weight': weights, 'synapse_model': syn_model})
+        nest.Connect(sources, targets, conn_spec='one_to_one', syn_spec={'weight': weights})
```

Comment:
The intention of this test is to check that `connect_arrays` works when specifying only weights. Now, in addition to that, it checks that connecting a list to a NumPy array works. It's probably best to check that in a separate test.